doc/gettingstarted.md

```diff
@@ -25,16 +25,46 @@ In this guide we will get everything up and running on a server using Docker.
 
 For now, the web interface Console is not available; in this tutorial, we use the command-line interface (CLI) to manage the network.
 
-Get the latest packages or binaries for your operating system from the [releases](https://github.com/TheThingsNetwork/lorawan-stack/releases) page on GitHub.
+In this tutorial the stack will be run inside a provided Docker container, and you should run the CLI commands from your local machine.
```

Comment:
`In this tutorial, the stack will run in Docker on the server, and you manage the stack either using the CLI on your local machine or on the server.`